### PR TITLE
first version of hibernate-relations homework

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

**Review by ArtemPonomar — APPROVED** [review]
Perfect

File: src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java [hunk 1]
```diff
@@ -1,9 +1,20 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+
```
Comment (nacenik) [inline src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java:10]: Empty line

File: src/main/resources/hibernate.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <property name="connection.username">root</property>
+        <property name="connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (nacenik) [inline src/main/resources/hibernate.cfg.xml:15]: Add empty line at the end of the file

**Review by nacenik — APPROVED** [review]
LGTM! :)

**Pull request closed without merging**